public/ipc.js

```diff
@@ -157,17 +157,33 @@ ipc.on('get job', async (event, data) => {
 })
 
 ipc.on('abort job', (event, data) => {
-  Object.values(queues[data.job.queue.name].childPool.retained).forEach((child) => {
-    child.send({__abortJobId__: data.job.id})
+
+  const dialogOptions = {type: 'info', buttons: ['Cancel', 'Abort'], message: 'Do you want to abort the current job?'}
```

Comment:
No need for a `dialogOptions` variable. Type `'question'` might be more apt here. Let's change the question to: `Are you sure you would like to abort {DESCRIPTION}?`, where `DESCRIPTION` is a short description for the task.